src/SiteMaster/Core/Auditor/Site/Page/ScanForm.php

```diff
@@ -166,7 +165,21 @@ public function handlePost($get, $post, $files)
      */
     protected function scan($get, $post, $files)
     {
-        $page_scan = Page::createNewPage($this->scan->id, $this->site->id, $this->uri, $this->page->found_with, array(
+        if (!$this->page && $this->scan->getDistinctPageCount() >= Config::get('SCAN_PAGE_LIMIT')) {
```

Comment:
I addressed this because I could see other potential uses for it, but currently the only other place where this code is used is here: https://github.com/UNLSiteMaster/site_master/blob/master/src/SiteMaster/Core/Auditor/Logger/Scheduler.php#L67

I didn't change that code because it would result in extra database calls which could slow things down.